integration-tests/site/_posts/2024-08-10-tips-to-improve-your-workflow.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: 10 Tips To Improve Your Workflow
+date: 2024-08-10 00:00:00 +0300
```

Comment:
Why not to have the date in that format yyyy-MM-dd, I think is more simple, WDYT?